extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -489,6 +490,16 @@ public void handle(RoutingContext event) {
                 }
             });
         }
+        httpRouteRouter.route().order(RouteConstants.ROUTE_ORDER_ROUTING_CONTEXT).handler(new Handler<RoutingContext>() {
```

Comment:
Hi @michalvavrik This PR is easy to understand and is totally fine, but I'd like to hear from @franz1981 and @cescoffier what do they think about the possible performance impact, cheers

Reply:
Sure, let's gather feedback.

FYI I can easily avoid creating handler by putting in for example here:

https://github.com/quarkusio/quarkus/blob/cdbb8e159aab00719a4fbef8fd32150eb304fea8/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpSecurityRecorder.java#L244

but I thought having `RoutingContext` in duplicated context could be potentially beneficial to others as well, so I added it right after an HTTP request processing started. If this view is not shared, I will move it.

Thanks

Reply:
After some thinking, avoiding extra handler is probably a good idea @sberyozkin , I'll address it.

Reply:
Done.